server/server.go

```diff
@@ -481,13 +481,7 @@ func (s *Server) writeHandler(w http.ResponseWriter, req *http.Request) {
 
 	currentIndex := s.raftServer.CommitIndex()
 	count := currentIndex - s.snapConf.lastIndex
-	if uint64(count) > s.snapConf.snapshotAfter {
```

Comment:
See commit message:
`server/server.go:469:1:warning: cyclomatic complexity 11 of
function (*Server).writeHandler() is high (> 10) (gocyclo)`

It's about the complexity of the function. Extracting this method reduces it under 10, but any kind of refactoring would have done the job as well.